src/common/cluster-store.ts

```diff
@@ -81,6 +81,11 @@ export interface ClusterModel {
   /** Metadata */
   metadata?: ClusterMetadata;
 
+  /**
+   * Labels for the catalog entity
+   */
+  labels?: Record<string, string>;
```

Comment:
Why do we need to store labels?

Reply:
As answered elsewhere so that additional labels can be set in the future, and so that we don't need to constantly recheck the old `WorkspacesStore` file.